### PR TITLE
Adding a note for the eventrouter image

### PR DESCRIPTION
**Pull request by alokstech**
Version(s):

4.14, 4.13, 4.12, 4.11

Issue:

https://issues.redhat.com/browse/OBSDOCS-608

Link to docs preview:

https://docs.openshift.com/container-platform/4.13/logging/log_collection_forwarding/cluster-logging-eventrouter.html#cluster-logging-eventrouter-deploy_cluster-logging-eventrouter

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@alokstech Please squash your commits to a single commit

**Comment by ShaunaDiaz** [comment]
@shreyasiddhartha PTAL

**Comment by ShaunaDiaz** [comment]
@alokstech Squash the commits, let @shreyasiddhartha take a look, then get QE. Once you have those three things, please put this back in for peer review. Thank you!

**Comment by alokstech** [comment]
Hi @shreyasiddhartha, Can you take a look?

i have squashed the commits into one. Let me know for anymore changes.

File: modules/cluster-logging-eventrouter-deploy.adoc [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
 Use the following steps to deploy the Event Router into your cluster. You should always deploy the Event Router to the `openshift-logging` project to ensure it collects events from across the cluster.
 
+[NOTE]
+====
+The Event Router image is not a part of the Cluster Logging Operator and it needs to be downloaded separately.
```
Comment (ShaunaDiaz) [inline modules/cluster-logging-eventrouter-deploy.adoc:13]:
```suggestion
The Event Router image is not a part of the Cluster Logging Operator and needs to be downloaded separately.
```

**Comment by briandooley** [comment]
@alokstech I'll discuss this with the logging docs team and get back to you about what to do with it.

**Comment by libander** [comment]
@alokstech - Once the change @ShaunaDiaz suggested is completed, this should be ready for merge review.

File: modules/cluster-logging-eventrouter-deploy.adoc (hunk 1 above)
Comment (abrennan89) [inline modules/cluster-logging-eventrouter-deploy.adoc:13]:
```suggestion
The Event Router image is not a part of the {clo} and must be downloaded separately.
```
Please use the attribute for the Operator here.

**Review by abrennan89 — COMMENTED** [review]
One comment. I also noticed in the description that only versions up to 4.14 are listed, but anything added in the `main` branch should also apply for 4.15 also.

**Comment by alokstech** [comment]
@ShaunaDiaz @abrennan89 I have made the changes suggested. Please review.

**Review by abrennan89 — COMMENTED** [review]
/lgtm

**Review by bergerhoffer — COMMENTED** [review]
LGTM for merging!

Just a note for next time though, the link to the preview in the description should be a link to the preview generated from the PR that shows your changes (https://70385--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/log_collection_forwarding/cluster-logging-eventrouter.html#cluster-logging-eventrouter-deploy_cluster-logging-eventrouter), not a link to the live version of the docs. Thanks!

**Pull request merged**